SuperBuild/External_ITK.cmake

```diff
@@ -3,17 +3,20 @@ IF(ITK_DIR)
   FIND_PACKAGE(ITK 5.0.0 REQUIRED PATHS ${ITK_DIR} NO_DEFAULT_PATH)
 
   MESSAGE(STATUS "Using ITK available at: ${ITK_DIR}")
-  
+
   # Copy libraries to CMAKE_RUNTIME_OUTPUT_DIRECTORY
   PlusCopyLibrariesToDirectory(${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${ITK_LIBRARIES})
- 
+
   SET (PLUS_ITK_DIR ${ITK_DIR} CACHE INTERNAL "Path to store itk binaries")
 ELSE()
+
+  SET (PLUS_ITK_VERSION_MAJOR 5)
+  SET (PLUS_ITK_VERSION_MINOR 0)
   # ITK has not been built yet, so download and build it as an external project
   SetGitRepositoryTag(
     itk
     "${GIT_PROTOCOL}://itk.org/ITK.git"
-    "v5.0rc02"
+    "v${PLUS_ITK_VERSION_MAJOR}.${PLUS_ITK_VERSION_MINOR}rc02"

```

Comment:
I wouldn't do it quite like this. For example, there is unlikely to be 5.1rc02 etc...
You could change it to another variable like PLUS_ITK_VERSION_PATCH

Reply:
Ok, changed.